docs/tables/aws_cloudtrail_trail_event.md

```diff
@@ -38,13 +39,52 @@ select
 from
   aws_cloudtrail_trail_event
 where
-  log_group_name = 'aws-cloudtrail-logs-013122550996-77246e11' and
-  not read_only
+  log_group_name = 'required-your-log-group-name'
```

Comment:
```suggestion
  log_group_name = 'aws-cloudtrail-log-group-name'
```

Since this table is for CloudTrail logs, we could update the example name/help text to be something less generic

Reply:
Fixed